wrappers/rust/icicle-core/src/msm/tests.rs

```diff
@@ -168,14 +166,20 @@ where
         for batch_size in batch_sizes {
             let mut points = C::generate_random_affine_points(test_size * batch_size);
             let mut scalars = vec![C::ScalarField::zero(); test_size * batch_size];
+
+            // add some zero points
+            for _ in 0..100 {
```

Comment:
same comment for the zeros

Reply:
On the one hand we can
On the other, I don't think users of a function called `generate_random_affine_points` expect zero points being sprinkled in. We're not promising cryptographic rng here or anything and this function shouldn't be used to create secure randomness but still...

Reply:
Ok, I just thought that if you need it more than once it's worth writing it once. You could wrap this function is a test util that accepts the probability of each point to be zeroed but you should decide if it makes sense to you. If not that's fine too.

Reply:
Yes, creating a separate test util function makes sense, will do